### ISSUE TITLE
Failed to generate a getter function for a mapping of mapping

### ISSUE THREAD

**Issue by ngthanhtrung** [Neufund/TypeChain#29]
A mapping of mapping would have a getter function with 2 parameters both named `index`, which causes a conflict.

```typescript
public example(
  index: BigNumber | string,
  index: BigNumber | number
): Promise<BigNumber> {
  return promisify(this.rawWeb3Contract.example, [
    index.toString(),
    index.toString()
  ]);
}
```

**Comment by krzkaczor** [comment]
Thanks for the issue. It's definitely a bug. PRs welcomed. I am not sure when exactly I will find time to work on this.

**Comment by ericwooley** [comment]
Having the same issue, awesome project though, just a little bit of manual massaging to get past this.

**Comment by krzkaczor** [comment]
@ngthanhtrung @ericwooley This issue is missing exact repro code but I think it should be fixed in: `0.1.4`. Check it out :)
